firestore-counter/POSTINSTALL.md

```diff
@@ -20,13 +20,13 @@ match /databases/{database}/documents/pages/{page} {
 
 #### Set up a scheduled function
 
-Review the [scheduled function documentation](https://firebase.google.com/docs/functions/schedule-functions) to set up a call to `${function:controller.url}` every minute. You may need to enable some APIs in your Firebase project to use scheduled functions.
+Review the [scheduled function documentation](https://firebase.google.com/docs/functions/schedule-functions) to set up a call to `${function:aggregator.url}` every minute. You may need to enable some APIs in your Firebase project to use scheduled functions.
 
 As an example, to set up a scheduled function, you can run the following [`gcloud`](https://cloud.google.com/sdk/gcloud/) commands:
```

Comment:
Revise to: 
You can set up a new Cloud Scheduler Job using the [Cloud Console](https://console.cloud.google.com/cloudscheduler/jobs/new) or by running the following [`gcloud`](https://cloud.google.com/sdk/gcloud/) commands: